charts/kcp/templates/kcp.yaml

```
@@ -247,6 +291,11 @@ spec:
         - --client-ca-file=/etc/kcp/tls/server-client/ca.crt
         - --tls-private-key-file=/etc/kcp/tls/server/tls.key
         - --tls-cert-file=/etc/kcp/tls/server/tls.crt
+        - --service-account-key-file=/etc/kcp/tls/service-account/tls.key
+        - --service-account-private-key-file=/etc/kcp/tls/service-account/tls.key
+        {{- if .Values.kcp.homeWorkspaces.enabled }}
+        - --enable-home-workspaces
```

This flag defaults to true so I think this will be a no-op and setting `kcp.homeWorkspaces.enabled=false` won't actually disable it. I suspect it should be more like this?

```suggestion
        - --enable-home-workspaces={{ .Values.kcp.homeWorkspaces.enabled }}
```